### ISSUE TITLE
Bullet-point lists are escaped when producing the md file

### ISSUE THREAD

**Issue by rgosse** [activeviam/react-api-documenter#35]
A list with hyphens has its new line characters removed:

```
- list
- of
- things
```
becomes
``` 
-list - of -things
```

Using the `*` character does not work either as it will be escaped when creating the md file.

**Comment by Nouzbe** [comment]
@rgosse I think you discovered that this is not as easily fixable as it seems. Can you add some details about your discovery here?

**Comment by rgosse** [comment]
Yes, `api-extractor` will strip new lines unless they are doubled, but doubled new lines will generate a `DocParagraph` which will make the bullet list appear too big.
Dylan had opened an issue about that to them some time ago:
https://github.com/microsoft/rushstack/issues/1441
